headers/types/ground_mode/enums.h

```diff
@@ -0,0 +1,1708 @@
+// Enums used primarily in the context of ground mode
+
+#ifndef HEADERS_TYPES_GROUND_MODE_ENUMS_H_
```

Comment:
If you want to be game-accurate it would probably be better to replicate the entire tables with all their settings (as an array of structs?)

You could keep the enums for easier lookup.

Reply:
Ah, I see you do have structs defined for the data. Don't know if you also want to consider adding arrays with the concrete data from the games in it or if that's out of scope.

Reply:
Do you mean like recording actual field values from the binaries? Yeah I think that's out of scope (also why I think this isn't a viable replacement for `pmd2scriptdata.xml`. I'd like to keep things at struct definitions without declaring any variables. If people want to know the actual data they can look in the binaries using the layout definitions, or look at `pmd2scriptdata.xml`.